### ISSUE TITLE
2016005 2016-03-11 (Georges Zangerlé)

### ISSUE THREAD

**Issue by rvdlaar8** [sdcTools/UserSupport#6]
Version used: 4.1.3
## Windows

In interactive mode kun je overlappende variabelen inlezen. Je krijgtwel een "warning", maar je kunt wel door gaan. In batch mode leidt het tot een error en stopt tau.
VB: (in rda-file)
SBI 58 4
<RECODEABLE>
<TOTCODE> "Total"
<HIERARCHICAL>
<HIERLEVELS> 1 2 1 0 0

SBI_long 58 5
<RECODEABLE>
<TOTCODE> "Total"
<HIERARCHICAL>
<HIERLEVELS> 1 2 1 1 0
Voorstel Georges:
korte termijn: vervang error door warning in batch mode
lange termijn: recoding in batch mode toevoegen (bij gebruik microdata), bv <RECODE> (Tablenumber, "ExpVar", Maxlevel, "GlobalRecodeFilename") en mogelijk ook aan rda-file.

**Comment by g3o2** [comment]
Please find here below the original issue in English.

When specifying overlapping variables in the `RDA` file, tauArgus behaves differently whether in interactive mode or in batch mode:

* in **interactive** mode, a message informs about the overlapping and clicking on `ok` allows to proceed;
* in **batch** mode, run from within tauArgus, an error is produced and further execution is interrupted. 

```xml
snarev2 58  4
    <RECODEABLE>
    <TOTCODE> "Total"
    <HIERARCHICAL>
    <HIERLEVELS>  1 2 1 0 0

snarev2_long 58  5
    <RECODEABLE>
    <TOTCODE> "Total"
    <HIERARCHICAL>
    <HIERLEVELS>  1 2 1 1 0
```

_In the short run_, one could drop the error message in batch mode and let execution continue while logging the error as a warning in the log file instead.

_In the long run_, in order to align with interactive mode features, one could introduce a `<RECODE> (TableNumber, "ExpVar", MaxLevel, "GlobalRecodeFilename")` command in batch file mode in addition to having such a feature in the RDA file. `MaxLevel` should default to `0`, suggesting that there should not be any recoding.

**Comment by ppdewolf** [comment]
Solved by commit db135c9 in tauargus repo.
